packages/unigraph-dev-explorer/package.json

```diff
@@ -6,20 +6,22 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "react": "^17.0.1",
+    "react-codemirror": "^1.0.0",
```

Comment:
hmmmmm I think react-codemirror is no longer maintained and is on react 15. can use it for now but long term I would want to migrate to another text editor library like [Slate](https://github.com/ianstormtaylor/slate)

Reply:
Agreed. For now we're just providing a simple interface for making developer queries but if we want to expose it to users we can definitely use Slate instead (it's also quite extensible too, which is perfect to our use case)